VaSeUtils/ygor_protocol.sh

```diff
@@ -0,0 +1,16 @@
+#string ygorLocation
+#string vbGoldStandard
+#string vbVcfCalls
+#string ygorOutdir
```

Comment:
We can probably use an existing parameter for this one, unless we plan on making a new directory within the CV runs to hold the results.